Sources/CoreData/CoreDataAccess.swift

```diff
@@ -138,8 +141,8 @@ public final class CoreDataAccess: DataAccess {
      - parameter operation: The actions to execute upon the background `DataContext`; will be passed said context.
      */
     public func performInBackground(operation: @escaping (_ context: DataContext) -> Void) {
-        container.performBackgroundTask { context in
-            context.mergePolicy = NSMergeByPropertyStoreTrumpMergePolicy
+        container.performBackgroundTask { [weak self] context in
+            context.mergePolicy = self?.defaultContextPolicy ?? .error
```

Comment:
Can you capture the `defaultContextPolicy` value as a local variable outside the scope of the block so you don't have to retain `self` and fallback on a nil-coalesced value that may not be the default (of `NSMergeByPropertyStoreTrumpMergePolicy`)?